Integrations/HelloWorld/HelloWorld.py

```diff
@@ -0,0 +1,185 @@
+            }
+        ]
+
+
+def test_module(client):
+    """
+    returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+    """
+    result = client.say_hello("DBot")
+    if "Hello DBot" == result:
+        return 'ok'
+    else:
+        return 'Test failed because ......'
+
+
+def say_hello_command(client, args):
+    name = args.get('name')
+
+    result = client.say_hello(name)
+
+    # readable output will be in markdown format - https://www.markdownguide.org/basic-syntax/
+    readable_output = "{}".format(result)
+    outputs = {
+        "hello": result
+    }
+
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def say_hello_over_http_command(client, args):
+    name = args.get("name")
+
+    result = client.say_hello_http_request(name)
+
+    # readable output will be in markdown format - https://www.markdownguide.org/basic-syntax/
+    readable_output = "## {}".format(result)
+    outputs = {
+        "hello": result
+    }
+
+    return (
+        readable_output,
+        outputs,
+        result  # raw response - the original response
+    )
+
+
+def fetch_incidents(client, last_run, first_fetch_time):
+    """
+    This function will execute each 1 minute.
+
+    :return: next_run, list of incidents that will be created in Demisto
+    """
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get('last_fetch')
+
+    # Handle first time fetch
+    if last_fetch is None:
+        last_fetch, _ = parse_date_range(first_fetch_time)
+    else:
+        last_fetch = datetime.strptime(last_fetch, DATE_FORMAT)
+
+    latest_created_time = last_fetch
+    incidents = []
+    items = client.list_incidents()
+    for item in items:
+        incident_created_time = datetime.strptime(item["created_time"], DATE_FORMAT)
+        incident = {
+            "name": item["description"],
+            "occurred": incident_created_time.strftime("%Y-%m-%dT%H:%M:%SZ"),
+            "rawJSON": json.dumps(item)
+        }
+
+        incidents.append(incident)
+
+        # Update last run and add incident if the incident is newer than last fetch
+        if incident_created_time > latest_created_time:
+            latest_created_time = incident_created_time
+
+    next_run = {"last_fetch": latest_created_time.strftime(DATE_FORMAT)}
+    return next_run, incidents
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    username = demisto.params().get('credentials').get('identifier')
+    password = demisto.params().get('credentials').get('password')
+
+    # get the service API url
+    base_url = urljoin(demisto.params()['url'], '/api/v1/suffix')
+
+    verify_certificate = not demisto.params().get('insecure', False)
+
+    # How many time before the first fetch to retrieve incidents
+    first_fetch_time = demisto.params().get('fetch_time', '3 days')
```

Comment:
need to strip this param incase a white space was entered (happened to us in the past)